Riot/Modules/Room/RoomViewController.m

```diff
@@ -351,11 +351,7 @@ - (void)viewDidLoad
     
     [self setupRemoveJitsiWidgetRemoveView];
     
-    // Replace the default input toolbar view.
-    // Note: this operation will force the layout of subviews. That is why cell view classes must be registered before.
-    dispatch_async(dispatch_get_main_queue(), ^{
-        [self updateRoomInputToolbarViewClassIfNeeded];
-    });
+    [self updateRoomInputToolbarViewClassIfNeeded];
```

Comment:
Looking at the [commit](https://github.com/vector-im/element-ios/pull/6565/commits/34d09cd0f9abac7294652cb877c9aca226c8ee87) where this change was made, I think the comment is a bit of a red herring as to why the dispatch is used. Seems that comment should have gone inside the block 🤔

Reply:
I reverted this change and, by applying the current theme to the `MXKRoomInputToolbarViewWithSimpleTextView`, I managed to cover somehow the glitch. Nevertheless, the glitch still exists.